src/functions-and-arrays.js

```diff
@@ -52,14 +151,47 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let newUniquifyArray = [];
+  let word = arr.length;
+
+  if (arr.length === 0) {
+    return null;
+  }
+
+  for (let i = 0; i < word; i++) {
+    if (newUniquifyArray.indexOf(arr[i]) === -1) {
+      newUniquifyArray.push(arr[i]);
+    }
+  }
+  return newUniquifyArray;
+}
+
+
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, searchWord) {
+  if (arr.length === 0) {
+    return null;
+  }
+
+  if (arr.includes(searchWord)) {
```

Comment:
Si le añadimos un else ya tenemos la función resuelta solo con esto
```suggestion
  if (arr.includes(searchWord)) {
    return true;
  } else {
    return false
  }
```